internal/apis/kfd/v1alpha2/eks/deleter.go

```diff
@@ -131,8 +134,10 @@ func (d *ClusterDeleter) Delete() error {
 			return fmt.Errorf("error while deleting kubernetes phase: %w", err)
 		}
 
-		if err := infra.Exec(); err != nil {
-			return fmt.Errorf("error while deleting infrastructure phase: %w", err)
+		if d.furyctlConf.Spec.Infrastructure != nil {
+			if err := infra.Exec(); err != nil {
+				return fmt.Errorf("error while deleting infrastructure phase: %w", err)
```

Comment:
didn't we already change these messages to something more user-friendly? and put in debug the phase thing like `Infrastructure: error while deleting resources: %w`

Reply:
yep but not in this file, we need to do that around all the codebase

Reply:
ok, but let's not introduce new ones that we'll need to fix later at least :)

Reply:
it's not a new one, it's the same shifted two lines below ^^"

Reply:
touché 😆